### ISSUE TITLE
Gifted skill using with items isn't working in NA

### ISSUE THREAD

**Issue by MeowyMeow** [Lambda11/pet-skill-replacer-and-user#16]
Opcode for C_START_SERVANT_ACTIVE_SKILL updated (56638). When partner isn't summoned it gives the regular error message (partner not found etc), but when partner is summoned using the item doesn't do anything - the item isn't used (which is the expected behavior) but the partner doesn't use the skill either. I tested both on partners that already had the skill I wanted to force to use, and partners that didn't have it.

**Comment by l0osee** [comment]
Yeah! By pressing the skill button it dosen't do nothing! The Opcode is correct but something is wrong
